WalletWasabi/WabiSabi/Client/DependencyGraphTaskScheduler.cs

```diff
@@ -181,11 +185,17 @@ public async Task StartOutputRegistrationsAsync(IEnumerable<TxOut> txOuts, BobCl
 					{
 						hdPubKey.SetKeyState(KeyState.Used);
 					}
+
+					throw;
```

Comment:
This will break the whole process and CoinJoinClient stops. I think this is wrong.

Reply:
- Output won't be registered and in the end, the sanity check will catch that problem - that is the concept. If you throw here, WhenAll will throw and it will bubble up to StartRoundAsync.

Reply:
Oh I see what you mean now. Yes, It is a mistake. I will fix it.